libpod/runtime_linux.go

```diff
@@ -0,0 +1,44 @@
+//go:build !remote
+
+package libpod
+
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+)
+
+const TMPFS_WARNING_ENV_VAR = "PODMAN_SUPPRESS_TMPFS_WARNING"
+
+// Print a warning if a path is not on a tmpfs.
+func warnIfNotTmpfs(paths []string) {
+	if os.Getenv(TMPFS_WARNING_ENV_VAR) != "" {
```

Comment:
I think we need to avoid the warning also when running in a container (the `container` env is set)